  application
  brick_admin:brick/0
brick_admin.erl:1311: The created fun only terminates with explicit exception
brick_admin.erl:1756: The created fun only terminates with explicit exception
brick_admin.erl:2363: The created fun only terminates with explicit exception
brick_admin_event_h.erl:112: The created fun only terminates with explicit exception
  brick_bp:brick_name/0
  brick_bp:nowtime/0
  brick_bp:proplist/0
  brick_bp:state_name/0
  brick_chainmon:brick_list/0
brick_ets.erl:1502: Function my_insert/6 has no local return
brick_ets.erl:1521: Function my_insert2/6 has no local return
brick_ets.erl:1607: Function my_delete/3 has no local return
brick_ets.erl:1836: Function checkpoint_start/4 only terminates with explicit exception
brick_ets.erl:1926: The created fun only terminates with explicit exception
brick_ets.erl:2349: The pattern {'reply', Reply, NewInterimS} can never match the type {'noreply',#state{name::atom(),options::'undefined' | [any()],start_time::'undefined' | {_,_,_},do_logging::boolean(),do_sync::boolean(),log_dir::'undefined' | [any()],bigdata_dir::'undefined' | [any()],n_add::non_neg_integer(),n_replace::non_neg_integer(),n_set::non_neg_integer(),n_get::non_neg_integer(),n_get_many::non_neg_integer(),n_delete::non_neg_integer(),n_txn::non_neg_integer(),n_do::pos_integer(),n_expired::non_neg_integer(),logging_op_serial::non_neg_integer(),logging_op_q::queue(),log::'undefined' | pid(),thisdo_mods::'undefined' | [any()],check_pid::'undefined' | pid(),check_lastseqnum::'undefined' | integer(),ctab::atom(),shadowtab::atom(),shadowtab_name::atom(),bypass_shadowtab::boolean(),etab::atom(),mdtab::atom(),sync_pid::'undefined' | pid(),checkpoint_timer::'undefined' | reference(),checkpoint_opts::'undefined' | [any()],dirty_tab::atom(),wait_on_dirty_q::'undefined' | queue(),read_only_p::boolean(),expiry_tref::'undefined' | reference(),scavenger_pid::'undefined' | pid(),scavenger_tref::'undefined' | reference(),syncsum_count::non_neg_integer(),syncsum_msec::non_neg_integer(),syncsum_len::integer(),syncsum_tref::'undefined' | reference(),do_expiry_fun::'undefined' | fun(),wal_mod::atom()}}
brick_ets.erl:2820: Guard test is_record(S::'undefined','state',44) can never succeed
brick_ets.erl:2953: The created fun has no local return
brick_ets.erl:2965: Function squidflash_doit/5 only terminates with explicit exception
brick_ets.erl:2970: The created fun only terminates with explicit exception
brick_ets.erl:2987: Function squidflash_prime1/5 only terminates with explicit exception
  brick_ets:extern_tuple/0
  brick_ets:state_r/0
  brick_hash:chash_r/0
  brick_hash:fixed_prefix_r/0
  brick_hash:naive_r/0
  brick_hash:var_prefix_r/0
brick_mboxmon.erl:318: The created fun has no local return
brick_mboxmon.erl:333: Function start_water_report_loop/3 has no local return
brick_mboxmon.erl:342: Function water_report_loop/2 only terminates with explicit exception
brick_mboxmon.erl:376: The created fun only terminates with explicit exception
brick_mboxmon.erl:379: The call erlang:register(ResumeName::atom(),pid()) might fail due to a possible race condition caused by its combination with the erlang:whereis(ResumeName::atom()) call in brick_mboxmon.erl on line 377
  brick_server:brick/0
  brick_server:brick_name/0
  brick_server:chain_name/0
  brick_server:delete_reply/0
  brick_server:flags_list_many/0
  brick_server:get_reply/0
  brick_server:global_hash_r/0
  brick_server:node_name/0
  brick_server:prop_list/0
  brick_server:repair_state_name/0
  brick_server:repair_state_name_base/0
  brick_server:role/0
  brick_server:set_reply/0
  brick_squorum:brick_list/0
  brick_squorum:set_res/0
cluster_info.erl:36: The specification for cluster_info:start/0 states that the function might also return {'error',_} | {'ok',pid(),_} but the inferred return is {'ok',pid()}
cluster_info.erl:37: The specification for cluster_info:start/2 states that the function might also return {'error',_} | {'ok',pid(),_} but the inferred return is {'ok',pid()}
  Compiling some key modules to native code... done in 0m45.50s
 done in 1m34.62s
done (warnings were emitted)
  erlang:now/0
  gen_event
  gen_fsm
  gen_server
gmt_cli.erl:166: Function close_exit_normal/1 only terminates with explicit exception
gmt_hlog_common.erl:1061: The created fun only terminates with explicit exception
gmt_hlog_common.erl:1185: The created fun only terminates with explicit exception
gmt_hlog_common.erl:192: The call erlang:register(CName::atom(),pid()) might fail due to a possible race condition caused by its combination with the erlang:whereis(CName::any()) call in gmt_hlog_common.erl on line 190
gmt_hlog_common.erl:316: The created fun only terminates with explicit exception
gmt_hlog_common.erl:748: The created fun only terminates with explicit exception
gmt_hlog_common.erl:952: The created fun only terminates with explicit exception
gmt_hlog_local.erl:384: The created fun only terminates with explicit exception
gmt_pmap.erl:99: The created fun only terminates with explicit exception
  gmt_tlog_printer:tlog_format/2
gmt_util.erl:1114: Function wait_for_processes_loop/2 only terminates with explicit exception
gmt_util.erl:560: Function shut_down_nicely/1 only terminates with explicit exception
mod_admin.erl:1170: The pattern [Nd | _] can never match the type []
  orddict:orddict/0
partition_detector.erl:29: The specification for partition_detector:start/0 states that the function might also return {'ok',pid(),_} but the inferred return is 'ignore' | {'error',_} | {'ok',pid()}
partition_detector.erl:30: The specification for partition_detector:start/2 states that the function might also return {'ok',pid(),_} but the inferred return is 'ignore' | {'error',_} | {'ok',pid()}
partition_detector_mnesia.erl:40: The specification for partition_detector_mnesia:init/1 states that the function might also return 'ignore' | {'stop',_} | {'ok',_,'infinity' | non_neg_integer()} but the inferred return is {'ok',{'state'}}
partition_detector_mnesia.erl:41: The specification for partition_detector_mnesia:handle_call/3 states that the function might also return {'noreply',_} | {'noreply',_,'timeout'} | {'stop',_,_} | {'reply',_,_,'infinity' | non_neg_integer()} | {'stop',_,_,_} but the inferred return is {'reply',_,_}
partition_detector_mnesia.erl:42: The specification for partition_detector_mnesia:handle_cast/2 states that the function might also return {'noreply',_,'timeout'} | {'stop',_,_} but the inferred return is {'noreply',_}
partition_detector_mnesia.erl:43: The specification for partition_detector_mnesia:handle_info/2 states that the function might also return {'noreply',_,'timeout'} | {'stop',_,_} but the inferred return is {'noreply',_}
partition_detector_server.erl:61: The specification for partition_detector_server:handle_call/3 states that the function might also return {'noreply',_} | {'noreply',_,'timeout'} | {'stop',_,_} | {'reply',_,_,'infinity' | non_neg_integer()} but the inferred return is {'reply',_,_} | {'stop','normal','ok',_}
partition_detector_server.erl:62: The specification for partition_detector_server:handle_cast/2 states that the function might also return {'noreply',_,'timeout'} | {'stop',_,_} but the inferred return is {'noreply',_}
partition_detector_server.erl:63: The specification for partition_detector_server:handle_info/2 states that the function might also return {'noreply',_,'timeout'} but the inferred return is {'noreply',_} | {'stop',{'node_list_change',_,_},{'state',_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_}}
partition_detector_server.erl:64: The specification for partition_detector_server:init/1 states that the function might also return 'ignore' | {'stop',_} | {'ok',_,'infinity' | non_neg_integer()} but the inferred return is {'ok',{'state','undefined' | [any()],'undefined' | integer(),'undefined' | integer(),boolean(),'undefined' | pid(),'undefined' | pid(),'undefined' | port(),'undefined' | timer:tref(),_,[],[],[],[],0,'undefined' | fun(() -> any()),_,_,'undefined' | char(),'undefined' | integer()}}
partition_detector_server.erl:688: The created fun has no local return
partition_detector_sup.erl:34: The specification for partition_detector_sup:init/1 states that the function might also return 'ignore' | {'error',_} but the inferred return is {'ok',{{'one_for_one',15,60},[{_,_,_,_,_,_},...]}}
  Proceeding with analysis...
  proc_socket_server:server_status/1
  supervisor
  timer:tref/0
Unknown behaviours (behaviour_info(callbacks) does not return any specs):
Unknown functions:
Unknown types:
